### PR TITLE
Fix Road to Roam assets

### PR DESCRIPTION
**Pull request by westim**
Hello!

I saw the [message on my js13k repository](https://github.com/westim/road-to-roam/issues/1) and I'm aware that my initial submission was broken due to misconfigured assets.

These latest changes should be working correctly and the updated ZIP should be under the 13,312 kB limit.

I definitely made several technical changes after the deadline other than the assets, so I have no issue with moving this game to the **Unfinished** category.

Thank you!

### CODE REVIEW

**Comment by alcore** [comment]
tl:dr: It's fine.

The full explanation: I looked through that commit history. Issues with paths tend to happen every year so we're lenient with those - and strictly speaking, a *cough* certain organizer *cough* should've asked you to resubmit a fixed package, either rejecting your game, or stalling. But this went through silently so something on our end failed.

The other things you changed fall either under project structure and are bordering on irrelevant to your ratings, or under bugfixes with relatively small code edits. 

This repo is precisely for allowing such *small* bugfixes while keeping the process transparent and public. That said, there's a penalty in that we only start accepting PRs no sooner than a week into voting, so that initial votes reflect those bugs (if they were noticeable at all).

That was yesterday. 

I'll accept the PR and deploy it, but I'll let your game stew in the Unfinished category for one more day as an extra penalty, so you'll probably miss out on a few additional votes.

**Comment by alcore** [comment]
Deployed and working now: https://dev.js13kgames.com/games/road-to-roam.

@end3r Mind grabbing this zip and deploying it on the main site as well?

**Comment by end3r** [comment]
Sure, where can I find the zip exactly?

**Comment by end3r** [comment]
Sure, done!
